src/main/java/com/project/psk/model/dto/User/UserInfoDTO.java

```diff
@@ -0,0 +1,17 @@
+package com.project.psk.model.dto.User;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+//this dto is used in retrieving information of existing users
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserInfoDTO {
```

Comment:
this class is the same as UserCreateDTO. Is there a point in having 2 separate ones

Reply:
yea, i was thinking whether or not to provide id in the UserInfoDTO, because id's are autogenerated and are not needed in the UserCreateDTO i guess, but then I removed the id from UserInfoDTO and they became the same thing, forgot to delete. but now that we've talked that id's probably should be provided in the UserInfoDTO, I'm thinking about leaving these two, because now they will be different (UserInfoDTO will have id, UserCreateDTO will not)